src/base/bittorrent/torrentinfo.cpp

```diff
@@ -63,7 +63,7 @@ TorrentInfo TorrentInfo::loadFromFile(const QString &path, QString &error)
 {
     error.clear();
     libt::error_code ec;
-    TorrentInfo info(new libt::torrent_info(Utils::String::toStdString(Utils::Fs::toNativePath(path)), ec));
+    TorrentInfo info(NativePtr(new libt::torrent_info(Utils::String::toStdString(Utils::Fs::toNativePath(path)), ec)));
```

Comment:
did you used to pass in a wide string here? now that you disabled wide strings, libtorrent expects utf8 strings (in case Qt has a toUtf8() or something on its file paths)